### PR TITLE
3 Page sizes added

### LINKED ISSUE #48: Support for page sizes: A4, Notebook, Journal, Foolscap
Issue #10 had mentioned a possible feature for supporting different page sizes. Although the issue was closed, it doesn't seem to have been implemented.

Lot of times there are specific page sizes needed.

So in the UI there could be an option to select the page size, maybe a dropdown with different presets like A4, Notebook, Journal, Foolscap; and an advanced option of manually stating the page size in cms.

### PR DESCRIPTION
**Pull request by iampratham29**
Fixes #48

### CODE REVIEW

File: js/PageSize.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function pgsize(value) {
+  if (value == 'notebook') {
+    document.getElementById('page-a').style.height = (11 * 400) / 8.5;
```
Comment (saurabhdaware) [inline js/PageSize.js]: Hi I guess you're supposed to add `+ 'px'` at the end to get that value in pixels

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
Hi I've added a comment below

File: js/PageSize.js (hunk 1 above)
Comment (saurabhdaware) [reply js/PageSize.js]: oh and page-a is class so you will have to replace `document.getElementById('page-a')` with `document.querySelector('.page-a')`

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
Oh ok I understood what is missing, so the function pgSize is undefined for the page because it is not included in the HTML. You can copy-paste that function into [js/app.mjs](https://github.com/saurabhdaware/text-to-handwriting/blob/master/js/app.mjs) file and it is supposed to work then

**Comment by iampratham29** [comment]
But I had defined it😧

**Comment by iampratham29** [comment]
Okay.. I'll do that again

**Comment by saurabhdaware** [comment]
Closing this since it will have lots of conflicts now 😅

**Pull request closed without merging**